web/pgadmin/browser/server_groups/servers/databases/static/js/database.js

```diff
@@ -557,11 +559,13 @@ define('pgadmin.node.database', [
         if (notify) {
           pgAdmin.Browser.notifier.confirm(
```

Comment:
use `confirmDelete`

Reply:
`confirmDelete` is a type of dialog. It doesn't literally mean that it should be used only for delete. Disconnect can use as well.